tests/tests.js

```diff
@@ -52,6 +53,66 @@ exports.defineAutoTests = function() {
                 done();
             });
         });
+
+        /*
+
+        Tests if the local copy is at the correct place and can be accessed via file plugin.
+        on android this works, as the files are copied to the PERSISTENT location:
+
+         /data/data/<app-id>/files
+
+        on iOS the files are copied to
+
+         /var/mobile/Applications/<UUID>/Library  (or .../Documents in compatibility mode)
+
+        but the persistent directory is .../Library/files.
+
+         */
+        function syncAndTest(appId, success, fail) {
+            var sync = ContentSync.sync({
+                id: appId,
+                type: 'local',
+                copyRootApp: true
+            });
+            sync.on('complete', function(localDataPath) {
+                var file = appId + '/index.html';
+                window.requestFileSystem(LocalFileSystem.PERSISTENT, 0, function(fs) {
```

Comment:
I don't like the idea of checking in a test that will pass on Android but not on iOS. Can you switch this to using:

```
var file = appId + '/index.html';
window.resolveLocalFileSystemURL("file://" + data. localDataPath + "/" + file, function(fileEntry) {
    expect(fileEntry).toBeDefined();
    success();
}, function(e){
    fail(file + ' should exist in local copy. Error code ' + e.code);
});
```

That should work for both platforms but I'm writing this code freestyle so please check. We can write another test using window.requestFileSystem when we resolve issue #83.